CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## [[NEXT]](https://github.com/iExecBlockchainComputing/iexec-commons-containers/releases/tag/vNEXT) 2023
```

Comment:
Shall we update the year now? As long as we add a line in the changelog section, it might look strange to keep 2023. as you like, it's a detail.

Reply:
fixed in 708543